index.html

```diff
@@ -236,6 +236,11 @@ <h1>NetJSONGraph.js Example Demos</h1>
           >Clustering</a
         >
       </div>
+      <div class="cards">
+        <a href="./examples/network-nodes.html" target="_blank"
```

Comment:
Let's maintain consistency, eg:
```suggestion
        <a href="./examples/netjsongraph-wifi-clients.html" target="_blank"
```